autogpt_platform/frontend/src/hooks/useCopyPaste.ts

```diff
@@ -90,10 +90,10 @@ export function useCopyPaste(getNextNodeId: () => string) {
                 if (oldToNewIdMap[node.id]) {
                   const nodeConnections = pastedEdges
                     .filter(
-                      (edge) =>
+                      (edge: Edge) =>
```

Comment:
shouldn't be necessary if `pastedEdges` is typed right, right?

Reply:
Didn’t look into it that far, maybe but it’s not complaining about those types

Reply:
i typed the whole thing